src/theory/fp/theory_fp_rewriter.cpp

```diff
@@ -204,7 +205,7 @@ namespace rewrite {
 
     if (node[0] > node[1]) {
       Node normal = NodeManager::currentNM()->mkNode(kind::FLOATINGPOINT_EQ,node[1],node[0]);
-      return RewriteResponse(REWRITE_DONE, normal);
+      return RewriteResponse(REWRITE_AGAIN_FULL, normal);
```

Comment:
This change should not be needed (there is no other rewrite that could apply and `node[0]`/`node[1]` are already rewritten (or just about to be rewritten) at that point.

Reply:
Right.